src/main/java/jenkins/branch/MultiBranchProject.java

```diff
@@ -567,6 +567,11 @@ protected View getWelcomeView() {
         return new MultiBranchProjectEmptyView(this);
     }
 
+    @Override
+    public View getView(String name) {
+        return getPrimaryView();
```

Comment:
Uh, what? Unless I am missing something, this is a :bug: since it prevents you from defining alternate views.